cloudify_aws/elb/resources/load_balancer.py

```diff
@@ -62,7 +64,7 @@ def properties(self):
             resources = self.client.describe_load_balancers(
                 Names=[self.resource_id])
         except (ClientError, ParamValidationError) as e:
-            self.logger.warn('Ignoring error: {0}'.format(str(e)))
+            self.logger.warn('Ignoring error: {0}'.format(text_type(e)))
```

Comment:
Same question. No change is necessary, just curious.

Reply:
Same as above